Details for the 20.8.2 patch (#287)
oasislabs/docs: src/operators/current-testnet-parameters.md

## Deployment Change Log

### 2020-07-02 (Latest)

> Patch upgrade. There is a critical vulnerability in tendermint [described
> here](https://github.com/tendermint/tendermint/blob/master/CHANGELOG.md#v0336).

Reviewer: ```suggestion
Patch upgrade. This update addresses a security vulnerability in Tendermint
[described here](https://github.com/tendermint/tendermint/blob/master/CHANGELOG.md#v0336).
```
